pdns/ws-auth.cc

```diff
@@ -430,6 +433,9 @@ static void fillZone(UeberBackend& B, const DNSName& zonename, HttpResponse* res
               /* if you ever want to update this comparison function,
                  please be aware that you will also need to update the conditions in the code merging
                  the records and comments below */
+              if (a.qname == b.qname && b.qtype == a.qtype) {
```

Comment:
To avoid repeating compares, I'd rather have a nested if else here

Reply:
Hi @omoerbeek 
I hope new changes will be good for you.